apps/fluent-tester/src/TestComponents/Common/TestProps.tsx

```diff
@@ -0,0 +1,22 @@
+import { Platform } from 'react-native';
+// Please read http://93days.me/testing-react-native-application/ to understand why we set accessibilityLabel here.
+
+/**
+ * Android uses accessibilityLabel to find elements in tests, while other platforms use testID.
```

Comment:
Nit: Let's get a bit more specific as to why Android uses accessibilityLabel (here, it's a bit confusing)

"The underlying Android test drivers use accessibilityLabel to find elements. However, all other platforms use testID. Because of this, we're adding the `testProps` function that takes in a unique identifier and returns the correct prop based on the platform (testID for Win32, Windows, macOS, iOS; accessibilityId for Android)"

The second part of your comment should be kept as well

Reply:
Sidenote: I wonder if there's opportunity here to make an upstream change to React Native on Android to set accessibiltiyLabel to testID if accessibilityLabel is nil. Given that React Native is investing on Appium / WebdriverIO, I think it might be a good opportunity.

Reply:
Updated comments.
@Saadnajmi I feel that might lead to unexpected accessibility issues (where the user wasn't intending to set accessibilityLabel on Android) I will look into it though